Upstream-Status: Pending

diff --git a/src/ext/yaml-cpp/emitterutils.cpp b/src/ext/yaml-cpp/emitterutils.cpp
index 589d34062..94f05ce14 100644
--- a/src/ext/yaml-cpp/emitterutils.cpp
+++ b/src/ext/yaml-cpp/emitterutils.cpp
@@ -1,4 +1,5 @@
 #include <algorithm>
+#include <cstdint>
 #include <iomanip>
 #include <sstream>
 #include <cstdint>
diff --git a/src/lib/rlc/utils.hpp b/src/lib/rlc/utils.hpp
index 818fc55ed..6f36f2588 100644
--- a/src/lib/rlc/utils.hpp
+++ b/src/lib/rlc/utils.hpp
@@ -10,6 +10,7 @@

 #include <cstdint>
 #include <cstring>
+#include <cstdint>
 #include <memory>
 #include <vector>
