
Upstream-Status: Pending

diff --git a/Makefile b/Makefile
index 1a2677f..3cae0b6 100644
--- a/Makefile
+++ b/Makefile
@@ -23,8 +23,7 @@ MODE+= -O3 -s -Wall -Wextra -pedantic # //////////      RELEASE
 
 EDITOR	?=	geany
 
-NETLINK_FLAGS_PATH := /usr/include/libnl3
-NETLINK_FLAGS := -I $(NETLINK_FLAGS_PATH)
+NETLINK_FLAGS := $(shell pkg-config --cflags libnl-3.0)
 NETLINK_LIBS_PATH := .
 NETLINK_LIBS := -L $(NETLINK_LIBS_PATH) -lnl-genl-3 -lnl-3
 
